src/runtime/refreshUtils.js

```diff
@@ -27,7 +27,7 @@ function getReactRefreshBoundarySignature(moduleExports) {
     return signature;
   }
 
-  for (const key in moduleExports) {
+  for (let key in moduleExports) {
```

Comment:
I think there's no need to switch to `let` from `const` since IE11 supports both?

Reply:
That was my thought as well! However, it appears that IE11 has one small caveat: https://caniuse.com/#feat=const

> Not supported in for-in and for-of loops

😞

Reply:
OH MY GOD IE WHY 🤦